lib/hets-rabbitmq-wrapper.rb

```diff
@@ -4,44 +4,55 @@
 require 'bunny'
 
 module HetsRabbitMQWrapper
-  def self.run
-    connection = Bunny.new
-    connection.start
+  # Delivers queues for messages and decision which queues should be subscribed
+  class Listener
+    def call
+      connection = Bunny.new
```

Comment:
This line can be put into a `initialize` method, but make `connection` an attribute `@connection`.